src/lib/Preview.js

```diff
@@ -613,27 +616,53 @@ class Preview extends EventEmitter {
         // Clear any existing retry timeouts
         clearTimeout(this.retryTimeout);
 
-        // Save reference to the currently shown file, if any
+        // Save reference to the currently shown file ID, if any
         const currentFileId = this.file ? this.file.id : undefined;
 
+        // Save reference to the currently shown file version ID, if any
+        const currentFileVersionId = this.file && this.file.file_version ? this.file.file_version.id : undefined;
```

Comment:
Is there a reason that the file_version ID needs to be nested? Could we have it alongside the ID like 

```
this.file = {
    id: 1000000
    file_version_id: 1000001
}
```

Reply:
Looking into this more, I see we get back from the file version API `sha1` and `type`. Are we planning on using those?

Reply:
I don't think we use them now, but this is more so we don't have to maintain a file format other than what the Box API returns us.